pkg/config/config.go

```diff
@@ -524,6 +528,9 @@ func (c *Config) IsAuthUISimplified() bool {
 	return c.Auth.UIConfig.RBAC == AuthRBACSimplified
 }
 
+func (c *Config) IsAuthenticationTypeAPI() bool {
+	return c.Auth.AuthenticationAPI.Endpoint != ""
+}
 func (c *Config) IsAuthTypeAPI() bool {
```

Comment:
should this be changed to `IsAuthorizationTypeAPI`?

Reply:
Didn't touch the Auth as part of this PR, given the fact that it may include some breaking changes such as configuration names